tensorflow_io/core/python/experimental/image_ops.py

```diff
@@ -208,3 +208,17 @@ def decode_jp2(contents, dtype=tf.uint8, name=None):
       A `Tensor` of type `uint8` and shape of `[height, width, 3]` (RGB).
     """
     return core_ops.io_decode_jpeg2k(contents, dtype=dtype, name=name)
+
+
+def decode_obj(contents, name=None):
+    """
+    Decode a Wavefront (obj) file into a float32 tensor.
+
+    Args:
+      contents: A `Tensor` of type `string`. 0-D. The Wavefront (obj) file.
```

Comment:
@yongtang I think
`contents: A Tensor of type string. 0-D.  Wavefront (obj) file.` 

be replaced with
`contents: A 0 dimensional Tensor of type string, i.e the content of the Wavefront (.obj) file.`